### ISSUE TITLE
Could not find or load main class com.iris.driver.reflex.generator.ReflexGenerator

### ISSUE THREAD

**Issue by AndrewX192** [arcus-smart-home/arcusplatform#17]
From ./gradlew startService
```
> Configure project :arcusplatform:tools
Building tools v2.1.0-SNAPSHOT
[ant:java] Error: Could not find or load main class com.iris.driver.reflex.generator.ReflexGenerator

> Task :platform:arcus-containers:driver-services:generateDriversReflexDB FAILED

FAILURE: Build failed with an exception.

* Where:
Build file '/home/wlnet/arcusplatform/platform/arcus-containers/driver-services/build.gradle' line: 112

* What went wrong:
Execution failed for task ':platform:arcus-containers:driver-services:generateDriversReflexDB'.
> Java returned: 1

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

Deprecated Gradle features were used in this build, making it incompatible with Gradle 5.0.
Use '--warning-mode all' to show the individual deprecation warnings.
See https://docs.gradle.org/4.10.2/userguide/command_line_interface.html#sec:command_line_warnings

BUILD FAILED in 1m 1s
118 actionable tasks: 40 executed, 78 up-to-date
```

I've been using the following hack to get around this, with no observed effect:
```
$ git diff platform/arcus-containers/driver-services/build.gradle
diff --git a/platform/arcus-containers/driver-services/build.gradle b/platform/arcus-containers/driver-services/build.gradle
index ea089af..44274cf 100644
--- a/platform/arcus-containers/driver-services/build.gradle
+++ b/platform/arcus-containers/driver-services/build.gradle
@@ -109,7 +109,7 @@ task generateDriversReflexDB {
 
    doLast {
 
-      ant.java (fork:true, failonerror: true, classname: "com.iris.driver.reflex.generator.ReflexGenerator") {
+      ant.java (fork:true, failonerror: false, classname: "com.iris.driver.reflex.generator.ReflexGenerator") {
          arg(value: '--output')
          arg(value: 'none')
          arg(value: '--version')

**Comment by AndrewX192** [comment]
Potential part of the issue

`21:56:33.694 [DEBUG] [org.gradle.api.internal.project.ant.AntLoggingAdapter] dropping /home/wlnet/arcusplatform/common/arcus-drivers/reflex-generator/build/libs/reflex-generator.jar from path as it doesn't exist`

**Comment by AndrewX192** [comment]
Fix is in #37
